test/serve_seeds_api.ts

```diff
@@ -0,0 +1,276 @@
+import supertest from 'supertest'
+import web3 from 'web3';
+
+import { MusicPlatformType } from '../src/types/platform';
+
+describe('Seeds API', () => {
+  let app: any;
+  const Web3 = new web3();
+
+  // TODO: rather stub out the permittedAdminAddresses() to contain this address
+  // instead of having to put the known address into .env
+  const testAdminWallet = {
+    address: '0x8eb97c37B0BDe7A09eA5b49D6D97cd57e10559ba',
+    privateKey: '0xe07cc69757e3b261ffeb70df20f832ae74da57e11dd440a5da75377abe8caefc',
+  }
+
+  const wallet = Web3.eth.accounts.privateKeyToAccount(testAdminWallet.privateKey)
+
+  // TODO: spin up a test DB instead of using development
```

Comment:
As mentioned, I could use some help with this. Alternatively, we skip it.

Reply:
skip for now i think?